airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -28,6 +28,7 @@
 import io.airbyte.db.jdbc.streaming.AdaptiveStreamingQueryConfig;
 import io.airbyte.integrations.base.Source;
 import io.airbyte.integrations.source.jdbc.AbstractJdbcSource;
+import io.airbyte.integrations.source.relationaldb.RelationalDbQueryUtils;
```

Comment:
Feel free to ignore but you can probably 

```java
import io.airbyte.integrations.source.relationaldb.RelationalDbQueryUtils.getFullyQualifiedTableName;
```

so you don't need to call the full method `class.method`

Reply:
Sadly there is a duplicated `getFullyQualifiedTableName` method in JdbcSourceAcceptanceTest but removing the full method wherever possible